### PR TITLE
Fix installs on Windows

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
windows did not like my fancy `node -e require`

### CODE REVIEW

File: npm/install-wrangler.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+const { install } = require("./binary");
+install()
```
Comment (ashleymichal) [inline npm/install-wrangler.js]: no EOL i guess makes gh grumpy

File: npm/uninstall-wrangler.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+const { uninstall } = require("./binary");
+uninstall()
```
Comment (ashleymichal) [inline npm/uninstall-wrangler.js]: same

**Comment by EverlastingBugstopper** [comment]
@ashleymichal i used the `-e` so i wouldn't have to break things out into separate files that are two lines long lol

File: npm/install-wrangler.js (hunk 1 above)
Comment (EverlastingBugstopper) [reply npm/install-wrangler.js]: gh can have a little eol as a treat

**Pull request merged**